v0/destinations/mailchimp/transform.js

```diff
@@ -1,107 +1,153 @@
+/* eslint-disable no-nested-ternary */
+/* eslint-disable no-param-reassign */
 const get = require("get-value");
-const axios = require("axios");
 const md5 = require("md5");
-const { EventType, MappedToDestinationKey } = require("../../../constants");
-const {
-  getEndpoint,
-  destinationConfigKeys,
-  subscriptionStatus,
-  validStatuses
-} = require("./config");
+const { isDefinedAndNotNull } = require("rudder-transformer-cdk/build/utils");
```

Comment:
Change the reference to V0/utils.

Reply:
@ItsSudip why?

Reply:
I think its more of a concern about this transition. Should we start using cdk utils, how should we migrate. What would be the startegy.

Reply:
For now my personal vote is to do the migration as a separate process rather than starting to use that abruptly in a destination

Reply:
Basically, these generic utils we see in the transformer are cloned into CDK. If someone is using them from CDK, we should continue assuming they're making an informed choice. When we completely move them over to CDK, we'll make the changes everywhere.

Reply:
We can use CDK utils here as the functionality is the same in both places. I asked for this change to separate things from CDK as we are not using CDK to implement this destination.